src/makeCalculator.js

```diff
@@ -38,6 +38,39 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+    },
+
+    subtract(value) {
+      this.result -= value;
+    },
+
+    multiply(value) {
+      this.result *= value;
+    },
+
+    divide(value) {
+      this.result /= value;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(operation, value) {
+      operation.bind(this)(value);
```

Comment:
```suggestion
      operation.call(this, value);
```
Call/apply methods are more relevant here as you immidiately call the function
https://www.w3schools.com/js/js_function_call.asp

Reply:
Thanks for the advice!